include/bbp/sonata/config.h

```diff
@@ -0,0 +1,149 @@
+/*************************************************************************
+ * Copyright (C) 2018-2021 Blue Brain Project
+ *                         Jonas Karlsson <jonas.karlsson@epfl.ch>
+ *                         Juan Hernando <juan.hernando@epfl.ch>
+ *
+ * This file is part of 'libsonata', distributed under the terms
+ * of the GNU Lesser General Public License version 3.
+ *
+ * See top-level COPYING.LESSER and COPYING files for details.
+ *************************************************************************/
+
+#pragma once
+
+#include <map>
+#include <memory>  // std::unique_ptr
+#include <set>
+#include <string>
+#include <vector>
+
+#include <bbp/sonata/edges.h>
+#include <bbp/sonata/nodes.h>
+
+#include "common.h"
+
+
+namespace bbp {
+namespace sonata {
+
+/**
+ * Stores pouplation-specific network information.
+ */
+struct SONATA_API PopulationProperties {
+    /**
+     * Population type
+     */
+    std::string type;
+
+    /**
+     * Path to the template HOC files defining the E-Mode
+     */
+    std::string biophysicalNeuronModelsDir;
+
+    /**
+     * Path to the directory containing the morphologies
+     */
+    std::string morphologiesDir;
+
+    /**
+     * Dictionary for alternate directory paths.
+     */
+    std::map<std::string, std::string> alternateMorphologyFormats;
```

Comment:
Why map and not `unordered_map` ?

Reply:
No reason at all, there is no need for any sort. I replaced them with `unordered_map`. Thanks!